client/ayon_core/pipeline/create/creator_plugins.py

```diff
@@ -22,6 +23,11 @@
 from .legacy_create import LegacyCreator
 
 
+if TYPE_CHECKING:
+    from .context import CreateContext, CreatedInstance, UpdateData
+    ProjectSettings = Dict[str, Any]  # typing alias
```

Comment:
We don't need it per se - I just liked it better with regards to clarity that it's ProjectSettings.

Preferably I'd explicitly make it a `TypedAlias` across the codebase so that `ProjectSettings` would be returned from the functions as well. So that my IDE can report me a warning if what I'm passing is e.g. not `ProjectSettings` but `WhateverSettings` (even though they might both be `Dict[str, Any]` I could then still differentiate between the two in type checks. However, I believe that more explicit aliasing came in Py 3.8 and we're still supporting older, e.g. Py3.7 in Maya 2022.

Reply:
>  and we're still supporting older, e.g. Py3.7 in Maya 2022.

So this shouldn't be here, or?

Reply:
This should work fine in Py 3.5+ - but a better typing would be using the `TypedAlias` class - but that's only 3.8+ unfortunately. When using `TypedAlias` linters will actually check against that particular alias name instead of the `Dict[str, Any]` making type hints even better.

Reply:
Removed alias with https://github.com/ynput/ayon-core/pull/366/commits/183100445f0d1e681704f5493cf2c4bb382933ed to move this PR forward.